-------------------------------------------------------------------------------
Test set: de.tum.in.test.integration.AstAssertionTest$AnyConditionalTestTests
-------------------------------------------------------------------------------
Tests run: 4, Failures: 4, Errors: 0, Skipped: 0, Time elapsed: 0.019 s <<< FAILURE! - in de.tum.in.test.integration.AstAssertionTest$AnyConditionalTestTests
de.tum.in.test.integration.AstAssertionTest$AnyConditionalTestTests.test_testHasAtNoAnyConditional_Success  Time elapsed: 0.002 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: 
Expected test 'testHasAtNoAnyConditional_Success' to be successful. Failure:

java.lang.AssertionError: Unwanted statement found:
- In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/no/ClassWithNoKindsOfConditionals.java:
- Conditional-Expression was found:
   - Between line 8 (column 22) and line 8 (column 64)
   - Between line 8 (column 42) and line 8 (column 63)
	at de.tum.in.test.testutilities.CustomConditions.generateFailure(CustomConditions.java:178)
	at de.tum.in.test.testutilities.CustomConditions.lambda$testSuccessRespectingDirectFailure$11(CustomConditions.java:151)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at de.tum.in.test.testutilities.CustomConditions.lambda$testSuccessRespectingDirectFailure$12(CustomConditions.java:149)
	at org.assertj.core.api.Condition.matches(Condition.java:161)
	at java.base/java.util.Optional.filter(Optional.java:218)
	at org.junit.platform.testkit.engine.Event.lambda$byPayload$0(Event.java:122)
	at org.assertj.core.api.Condition.matches(Condition.java:161)
	at org.assertj.core.condition.AllOf.lambda$matches$0(AllOf.java:63)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
Caused by: java.lang.AssertionError: Unwanted statement found:
- In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/no/ClassWithNoKindsOfConditionals.java:
- Conditional-Expression was found:
   - Between line 8 (column 22) and line 8 (column 64)
   - Between line 8 (column 42) and line 8 (column 63)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at de.tum.in.test.integration.testuser.AstAssertionUser$AnyConditionalTests.testHasAtNoAnyConditional_Success(AstAssertionUser.java:339)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at de.tum.in.test.api.internal.TimeoutUtils.rethrowThrowableSafe(TimeoutUtils.java:47)
	at de.tum.in.test.api.internal.TimeoutUtils.lambda$performTimeoutExecution$1(TimeoutUtils.java:42)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

de.tum.in.test.integration.AstAssertionTest$AnyConditionalTestTests.test_testHasBelowNoAnyConditional_Fail  Time elapsed: 0.003 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: 

expected: 
  "Unwanted statement found:
   - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
    - If-Statement was found:
     - Between line 9 (column 3) and line 15 (column 3)
     - Between line 11 (column 10) and line 15 (column 3)
    - Switch-Statement was found:
     - Between line 20 (column 3) and line 30 (column 3)"
 but was: 
  "Unwanted statement found:
   - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
  - If-Statement was found:
     - Between line 9 (column 3) and line 15 (column 3)
     - Between line 11 (column 10) and line 15 (column 3)
  - Switch-Statement was found:
     - Between line 20 (column 3) and line 30 (column 3)"
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at de.tum.in.test.testutilities.CustomConditions.lambda$createMessageCondition$5(CustomConditions.java:88)
	at de.tum.in.test.testutilities.CustomConditions.lambda$createMessageCondition$7(CustomConditions.java:98)
	at org.junit.platform.commons.util.FunctionUtils.lambda$where$0(FunctionUtils.java:49)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1602)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	Suppressed: org.opentest4j.AssertionFailedError: Test 'testHasBelowNoAnyConditional_Fail' failed in an unexpected way:

TestExecutionResult [status = FAILED, throwable = java.lang.AssertionError: Unwanted statement found:
 - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
- If-Statement was found:
   - Between line 9 (column 3) and line 15 (column 3)
   - Between line 11 (column 10) and line 15 (column 3)
- Switch-Statement was found:
   - Between line 20 (column 3) and line 30 (column 3)]

Expectations:
[✗] all of:[
   [✓] instance of java.lang.AssertionError,
   [✓] message is correctly localized,
   [✗] message is 'Unwanted statement found:
 - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
  - If-Statement was found:
   - Between line 9 (column 3) and line 15 (column 3)
   - Between line 11 (column 10) and line 15 (column 3)
  - Switch-Statement was found:
   - Between line 20 (column 3) and line 30 (column 3)'
]
		at de.tum.in.test.testutilities.CustomConditions.generateFailure(CustomConditions.java:178)
		at de.tum.in.test.testutilities.CustomConditions.generateFailure(CustomConditions.java:173)
		at de.tum.in.test.testutilities.CustomConditions.lambda$testFailRespectingDirectFailure$8(CustomConditions.java:124)
		at java.base/java.util.Optional.ifPresentOrElse(Optional.java:196)
		at de.tum.in.test.testutilities.CustomConditions.lambda$testFailRespectingDirectFailure$10(CustomConditions.java:114)
		at org.assertj.core.api.Condition.matches(Condition.java:161)
		at java.base/java.util.Optional.filter(Optional.java:218)
		at org.junit.platform.testkit.engine.Event.lambda$byPayload$0(Event.java:122)
		at org.assertj.core.api.Condition.matches(Condition.java:161)
		at org.assertj.core.condition.AllOf.lambda$matches$0(AllOf.java:63)
	Caused by: java.lang.AssertionError: Unwanted statement found:
 - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
- If-Statement was found:
   - Between line 9 (column 3) and line 15 (column 3)
   - Between line 11 (column 10) and line 15 (column 3)
- Switch-Statement was found:
   - Between line 20 (column 3) and line 30 (column 3)
		at java.base/java.util.Optional.ifPresent(Optional.java:178)
		at de.tum.in.test.integration.testuser.AstAssertionUser$AnyConditionalTests.testHasBelowNoAnyConditional_Fail(AstAssertionUser.java:332)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:568)
		at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
		at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
		at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
		at de.tum.in.test.api.internal.TimeoutUtils.rethrowThrowableSafe(TimeoutUtils.java:47)
		at de.tum.in.test.api.internal.TimeoutUtils.lambda$performTimeoutExecution$1(TimeoutUtils.java:42)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
		at java.base/java.lang.Thread.run(Thread.java:833)

de.tum.in.test.integration.AstAssertionTest$AnyConditionalTestTests.test_testHasAtNoAnyConditional_Fail  Time elapsed: 0.002 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: 

expected: 
  "Unwanted statement found:
   - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
    - If-Statement was found:
     - Between line 9 (column 3) and line 15 (column 3)
     - Between line 11 (column 10) and line 15 (column 3)
    - Switch-Statement was found:
     - Between line 20 (column 3) and line 30 (column 3)"
 but was: 
  "Unwanted statement found:
  - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
  - If-Statement was found:
     - Between line 9 (column 3) and line 15 (column 3)
     - Between line 11 (column 10) and line 15 (column 3)
  - Switch-Statement was found:
     - Between line 20 (column 3) and line 30 (column 3)"
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at de.tum.in.test.testutilities.CustomConditions.lambda$createMessageCondition$5(CustomConditions.java:88)
	at de.tum.in.test.testutilities.CustomConditions.lambda$createMessageCondition$7(CustomConditions.java:98)
	at org.junit.platform.commons.util.FunctionUtils.lambda$where$0(FunctionUtils.java:49)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1602)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	Suppressed: org.opentest4j.AssertionFailedError: Test 'testHasAtNoAnyConditional_Fail' failed in an unexpected way:

TestExecutionResult [status = FAILED, throwable = java.lang.AssertionError: Unwanted statement found:
- In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
- If-Statement was found:
   - Between line 9 (column 3) and line 15 (column 3)
   - Between line 11 (column 10) and line 15 (column 3)
- Switch-Statement was found:
   - Between line 20 (column 3) and line 30 (column 3)]

Expectations:
[✗] all of:[
   [✓] instance of java.lang.AssertionError,
   [✓] message is correctly localized,
   [✗] message is 'Unwanted statement found:
 - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
  - If-Statement was found:
   - Between line 9 (column 3) and line 15 (column 3)
   - Between line 11 (column 10) and line 15 (column 3)
  - Switch-Statement was found:
   - Between line 20 (column 3) and line 30 (column 3)'
]
		at de.tum.in.test.testutilities.CustomConditions.generateFailure(CustomConditions.java:178)
		at de.tum.in.test.testutilities.CustomConditions.generateFailure(CustomConditions.java:173)
		at de.tum.in.test.testutilities.CustomConditions.lambda$testFailRespectingDirectFailure$8(CustomConditions.java:124)
		at java.base/java.util.Optional.ifPresentOrElse(Optional.java:196)
		at de.tum.in.test.testutilities.CustomConditions.lambda$testFailRespectingDirectFailure$10(CustomConditions.java:114)
		at org.assertj.core.api.Condition.matches(Condition.java:161)
		at java.base/java.util.Optional.filter(Optional.java:218)
		at org.junit.platform.testkit.engine.Event.lambda$byPayload$0(Event.java:122)
		at org.assertj.core.api.Condition.matches(Condition.java:161)
		at org.assertj.core.condition.AllOf.lambda$matches$0(AllOf.java:63)
	Caused by: java.lang.AssertionError: Unwanted statement found:
- In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/yes/ClassWithAnyKindsOfConditionals.java:
- If-Statement was found:
   - Between line 9 (column 3) and line 15 (column 3)
   - Between line 11 (column 10) and line 15 (column 3)
- Switch-Statement was found:
   - Between line 20 (column 3) and line 30 (column 3)
		at java.base/java.util.Optional.ifPresent(Optional.java:178)
		at de.tum.in.test.integration.testuser.AstAssertionUser$AnyConditionalTests.testHasAtNoAnyConditional_Fail(AstAssertionUser.java:346)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:568)
		at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
		at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
		at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
		at de.tum.in.test.api.internal.TimeoutUtils.rethrowThrowableSafe(TimeoutUtils.java:47)
		at de.tum.in.test.api.internal.TimeoutUtils.lambda$performTimeoutExecution$1(TimeoutUtils.java:42)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
		at java.base/java.lang.Thread.run(Thread.java:833)

de.tum.in.test.integration.AstAssertionTest$AnyConditionalTestTests.test_testHasBelowNoAnyConditional_Success  Time elapsed: 0.001 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: 
Expected test 'testHasBelowNoAnyConditional_Success' to be successful. Failure:

java.lang.AssertionError: Unwanted statement found:
 - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/no/ClassWithNoKindsOfConditionals.java:
- Conditional-Expression was found:
   - Between line 8 (column 22) and line 8 (column 64)
   - Between line 8 (column 42) and line 8 (column 63)
	at de.tum.in.test.testutilities.CustomConditions.generateFailure(CustomConditions.java:178)
	at de.tum.in.test.testutilities.CustomConditions.lambda$testSuccessRespectingDirectFailure$11(CustomConditions.java:151)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at de.tum.in.test.testutilities.CustomConditions.lambda$testSuccessRespectingDirectFailure$12(CustomConditions.java:149)
	at org.assertj.core.api.Condition.matches(Condition.java:161)
	at java.base/java.util.Optional.filter(Optional.java:218)
	at org.junit.platform.testkit.engine.Event.lambda$byPayload$0(Event.java:122)
	at org.assertj.core.api.Condition.matches(Condition.java:161)
	at org.assertj.core.condition.AllOf.lambda$matches$0(AllOf.java:63)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
Caused by: java.lang.AssertionError: Unwanted statement found:
 - In src/test/java/de/tum/in/test/integration/testuser/subject/structural/astTestFiles/conditionals/no/ClassWithNoKindsOfConditionals.java:
- Conditional-Expression was found:
   - Between line 8 (column 22) and line 8 (column 64)
   - Between line 8 (column 42) and line 8 (column 63)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at de.tum.in.test.integration.testuser.AstAssertionUser$AnyConditionalTests.testHasBelowNoAnyConditional_Success(AstAssertionUser.java:324)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at de.tum.in.test.api.internal.TimeoutUtils.rethrowThrowableSafe(TimeoutUtils.java:47)
	at de.tum.in.test.api.internal.TimeoutUtils.lambda$performTimeoutExecution$1(TimeoutUtils.java:42)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

